feat: get broker booking partner identifiers from config (#546)
openactive/openactive-test-suite: packages/openactive-broker-microservice/README.md

  }
```

> However, when running Broker Microservice as a standalone app, you may define any number of Booking Partners, with whatever identifiers you like e.g. `acme1`, `acme2` and `acme3` instead of `primary` and `secondary`. Just know that these will NOT work with the [Integration Tests](../openactive-integration-tests/), which expect only `primary` and `secondary`.

Reviewer: Consistent impersonal tone with rest of doc

```suggestion
When running Broker Microservice as a standalone app, any number of Booking Partners may be defined e.g. `acme1`, `acme2` and `acme3` instead of `primary` and `secondary`. Note that these will _not_ work with the [Integration Tests](../openactive-integration-tests/), which expect only `primary` and `secondary`.
```
